src/sagemaker/jumpstart/cache.py

```diff
@@ -338,29 +340,33 @@ def _retrieval_function(
             return JumpStartCachedContentValue(
                 formatted_content=model_specs
             )
-        if data_type == HubDataType.MODEL:
+        if data_type == HubContentType.MODEL:
             hub_name, region, model_name, model_version = utils.extract_info_from_hub_content_arn(
                 id_info
             )
-            hub = CuratedHub(hub_name=hub_name, region=region)
-            hub_content = hub.describe_model(model_name=model_name, model_version=model_version)
+            hub_model_description: HubContentDescription = hub_utils.describe_model(
```

Comment:
This should be renamed to `HubContent` or `HubContentResponse`. `HubContentDescription` doesn't map to anything in the [response](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_DescribeHubContent.html)